### DISCUSSION TITLE
Changing Configuration for Default Run Command

### DISCUSSION THREAD

**Discussion by Kakapio** [nextest-rs/nextest#1598] [General]
We currently use this command `cargo nextest run -E 'not package(regression-tests)' --no-fail-fast`
but I would like to have that configuration run automatically upon calling `cargo nextest run`

Is there any way to do this?  I haven't been able to find any documentation. There's a good bit about filtersets but I've found nothing about changing default run configuration.

**Comment by sunshowers** [comment]
Hi, thanks for the question! At the moment this isn't supported, but but it seems like a useful thing to have!

**Comment by sunshowers** [comment]
This is now supported. https://nexte.st/docs/running/#running-a-subset-of-tests-by-default

**Reply by Kakapio** [reply]
Thank you! You're awesome.
